src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -827,9 +823,11 @@ type AdaptiveFSharpLspServer(workspaceLoader: IWorkspaceLoader, lspClient: FShar
           |> Array.distinctBy (fun e ->
             e.Severity, e.ErrorNumber, e.StartLine, e.StartColumn, e.EndLine, e.EndColumn, e.Message)
 
-        let uri = Path.LocalPathToUri(file.Lines.FileName)
-        let diags = errors |> Array.map fcsErrorToDiagnostic
-        diagnosticCollections.SetFor(uri, "F# Compiler", diags)
+        NotificationEvent.ParseError(errors, file.Lines.FileName)
+        |> notifications.Trigger
```

Comment:
`ParseError` just sets the diagnostics collection the same way so reusing the event.